### PR TITLE
Improved documentation for classes modified by main window restructure.

### PR DESCRIPTION
**Pull request by Fincap**
This merge improves the comments and doc strings present within many of the new files added or modified by the main page restructure, as well as various miscellaneous files. This is by no means exhaustive, however in depth explanations of the new main page structure have been provided in order to explain not only what classes do, but also how they should be used.

### CODE REVIEW

File: src/Model/ImageLoading.py [hunk 1]
```diff
@@ -1,11 +1,4 @@
 """
-Skeleton for the interface between new boot window and existing patient window.
-Intention is to replace, recreate, or reuse most of the functionality served by the ProgressBar.Extended class.
-The functions in this file should be enough to generate the arguments required to create a MainWindow class.
-read_data_dict, file_names_dict, rois, raw_dvh, dvh_x_y, dict_raw_contour_data, dict_numpoints, dict_pixluts.
-The reason why some of these functions will be directly copied is because they currently exist as member functions
-of ProgressBar.Extended class and that class is to be deprecated/refactored with the new patient selection window.
-
```
Comment (Fincap) [inline src/Model/ImageLoading.py:8]: This section has been removed as it is no longer relevant to the current state of the file.

File: src/Model/ROI.py [hunk 2]
```diff
@@ -1,29 +1,4 @@
...
-)
-from PyQt5.QtWidgets import (
-    QApplication,
-    QGraphicsScene,
-    QGraphicsView,
-    QLabel,
-    QMainWindow,
-    QVBoxLayout,
-    QWidget,
-)
```
Comment (Fincap) [inline src/Model/ROI.py:26]: Removed unused imports, snuck its way into one of the commits.

**Comment by lw803** [comment]
The added docstrings have made it the purpose of each function or class easy to digest, whilst still be concise. These will allow new team members to understand the code without having an intimiate knowledge of it's interactions.

**Review by lw803 — COMMENTED** [review]
My review is within the conversation of these changes already.

**Pull request merged**